pkg/controller/controller_test.go

```diff
@@ -460,17 +460,23 @@ const alphaParameterSchemaCatalogBytes = `{
       	"d": "e"
       },
       "schemas": {
-      	"service_instances": {
+      	"service_instance": {
 	  	  "create": {
-	  		"foo": "bar"
+	  	  	"parameters": {
+	  		  "foo": "bar"	
+	  	  	}
 	  	  },
 	  	  "update": {
-	  		"baz": "zap"
+	  	  	"parameters": {
+	  		  "baz": "zap"
+	  	    }
 	  	  }
       	},
-      	"service_bindings": {
+      	"service_binding": {
       	  "create": {
-      	 	"zoo": "blu"
+	  	  	"parameters": {
```

Comment:
weird that the formatting looks off via the web, but looking at the code its ok.